test/gurka/test_indoor.cc

```diff
@@ -51,6 +51,10 @@ class Indoor : public ::testing::Test {
         {"GH", {{"highway", "elevator"}, {"indoor", "yes"}, {"level", "1;2"}}},
         {"HI", {{"highway", "corridor"}, {"indoor", "yes"}, {"level", "2"}}},
         {"IJ", {{"highway", "corridor"}, {"indoor", "yes"}, {"level", "3"}}},
+
+        {"Cx", {{"highway", "steps"}, {"indoor", "yes"}, {"level", "1;2"}}},
+        {"xy", {{"highway", "steps"}, {"indoor", "yes"}, {"level", "2;3"}}},
+        {"yJ", {{"highway", "steps"}, {"indoor", "yes"}, {"level", "3"}}},
```

Comment:
i added an alternate path here, you can take the steps rather than use the elevator to get to the 3rd floor